config/sidebar.yml

```diff
@@ -338,7 +338,7 @@ libraries:
     children:
 
       - title: "Auth0.js v8"
-        url: "/libraries/auth0js"
+        url: "/libraries/auth0js/v8"
```

Comment:
Without this change the Libraries > Auth0 SDK for Web menu click was giving an error. Add this to the issue you'll create at auth0-docs.